conda/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+export PATH="$HOME/miniconda2/bin:$PATH"
+
+echo "Clearing out conda-bld"
+rm -rf /root/miniconda2/conda-bld
+rm -rf /root/miniconda2/pkgs
+
+echo "Rebuilding conda-bld"
+mkdir -p /root/miniconda2/conda-bld/linux-64
+mkdir -p /root/miniconda2/conda-bld/noarch
+conda index /root/miniconda2/conda-bld/linux-64
+conda index /root/miniconda2/conda-bld/noarch
+
+echo "Adding channels"
+conda config --remove channels defaults
```

Comment:
The only reason we did this is because we had a bunch of deps and the order actually mattered. If yours doesn't, then you don't really have to worry about removing and re-adding. Certainly can't hurt, though.